File: IRGCRM/app/src/main/java/web_api/APIClient.java
package web_api;

import com.google.gson.Gson;
import com.google.gson.GsonBuilder;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class APIClient {

    private static Retrofit retrofit = null;

    //http://localhost/irg_crm/api/userLogin.php?email_id=test@test.com&password=12345
    //http://192.168.0.100/irg_crm/api/userLogin.php?email_id=test@test.com&password=123456

  //  private static final String baseUrl="http://10.0.2.2/irg_crm/";
   private static final String baseUrl="http://192.168.0.105/irg_crm/";
    public static Retrofit getClient() {

        Retrofit retrofit = new Retrofit.Builder()
                .baseUrl(baseUrl)
                .addConverterFactory(GsonConverterFactory.create())
                .build();

        APIInterface service = retrofit.create(APIInterface.class);
        return retrofit;
    }
}

File: SWEEP/app/src/main/java/com/csows/sweep/controller/ApiClient.kt
package com.csows.sweep.controller

import hu.akarnokd.rxjava3.retrofit.RxJava3CallAdapterFactory
import okhttp3.OkHttpClient
import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory
import java.util.concurrent.TimeUnit


class ApiClient {

    companion object {
        private const val BASE_URL = "http://164.100.72.248/CeoDelhiOffice/api/"

         fun getClient(): APIInterface {

             val httpClient = OkHttpClient.Builder()
                 .callTimeout(2, TimeUnit.MINUTES)
                 .connectTimeout(20, TimeUnit.SECONDS)
                 .readTimeout(30, TimeUnit.SECONDS)
                 .writeTimeout(30, TimeUnit.SECONDS)

            var retrofit =  Retrofit.Builder()
                .baseUrl(BASE_URL)
                .addConverterFactory(GsonConverterFactory.create())
                .addCallAdapterFactory(RxJava3CallAdapterFactory.create())
                .client(httpClient.build())
                .build()

             val apiInterface : APIInterface = retrofit.create(APIInterface::class.java)

             return apiInterface
        }
        /*   private val retrofit = Retrofit.Builder()
               .baseUrl(BASE_URL)
               .addConverterFactory(GsonConverterFactory.create())
               .addCallAdapterFactory(RxJava3CallAdapterFactory.create())
               .build()

           val apiInterface : ApiInterface = retrofit.create(ApiInterface::class.java)*/
    }

}